src/Sensors/JobAttemptSensor.php

```diff
@@ -54,7 +54,7 @@ public function __invoke(JobProcessed|JobReleasedAfterException|JobFailed $event
             'trace_id' => $this->commandState->trace,
             'user' => $this->commandState->user->id(),
             // --- //
-            'job_id' => $event->job->uuid(),
+            'job_id' => $event->job->payload()['nightwatch']['job_id'] ?? $event->job->uuid(),
```

Comment:
Leaving the fallback in place here for those who upgrade from a lower version, so that jobs on the queue do not end up in a weird state.